pkg/storage/cacher_test.go

```diff
@@ -74,7 +74,7 @@ func waitForUpToDateCache(cacher *storage.Cacher, resourceVersion uint64) error
 	return wait.Poll(10*time.Millisecond, 100*time.Millisecond, ready)
 }
 
-func TestList(t *testing.T) {
+func TestListFromMemory(t *testing.T) {
```

Comment:
The new List() function is just delegating its work to underlying storage. I can add a test for it, but I'm not sure it adds any value.